firebase_test.go

```diff
@@ -350,6 +568,34 @@ func (t *testTokenSource) Token() (*oauth2.Token, error) {
 	}, nil
 }
 
+func overwriteEnv(varName, newVal string, t *testing.T) (string, bool) {
+	oldVal, wasSet := os.LookupEnv(varName)
+	if err := os.Setenv(varName, newVal); err != nil {
+		t.Fatal(err)
+	}
+	return oldVal, wasSet
+}
+
+func reinstateEnv(varName, oldVal string, varSet bool) {
+	if varSet {
+		os.Setenv(varName, oldVal)
+	} else {
+		os.Unsetenv(varName)
+	}
+}
+
+func compareConfig(got *App, want *Config, t *testing.T) {
+	if got.databaseURL != want.DatabaseURL {
+		t.Errorf("app.databaseURL = %q; want %q", got.databaseURL, want.DatabaseURL)
```

Comment:
Super nitpicky, but we've been using "want = %q" in our other tests instead of "want %q".

Reply:
This file has no such instances, would you like me to change the entire file (12 other instances)
(in other news I just changed wanted => want in this file.)